lib/org/openpsa/sales/config/routes.inc

```diff
@@ -118,9 +118,10 @@
     'handler' => array('org_openpsa_sales_handler_list', 'list'),
 ),
 
-// Match salesproject/action/create_pdf/
+// Match salesproject/action/<guid>/
 'create_pdf' => array
 (
-    'handler' => array('org_openpsa_sales_handler_sales_action', 'create_pdf'),
-    'fixed_args' => array('salesproject', 'action', 'create_pdf'),
+    'handler' => array('org_openpsa_sales_handler_action', 'create_pdf'),
+    'fixed_args' => array('salesproject', 'action'),
```

Comment:
It would be good if the url fragment gives you a clue as to its function, i.e. `action` is a bit generic when it's really a PDF renderer